django_airavata/apps/api/views.py

```diff
@@ -1535,7 +1534,7 @@ def put(self, request, path="/", format=None):
             data_products_helper.update_file_content(request=request, path=path,
                                                      fileContentText=request.data["fileContentText"])
         else:
-            return Response(status=HTTP_403_FORBIDDEN)
+            return Response(status=HTTP_404_NOT_FOUND)
```

Comment:
I see you changed this to 404, but really it should be 400 since the request is missing required values.

Reply:
Done